foreman/Dockerfile.tests

```diff
@@ -0,0 +1,21 @@
+# This Dockerfile is meant to be run from the directory above where it resides.
```

Comment:
Is this true? It seems to be setting `WORKDIR` using a full path manually at line 6... maybe I'm missing something?